tgui/packages/tgui/styles/themes/spookyconsole.scss

```diff
@@ -0,0 +1,64 @@
+@use 'sass:color';
+@use 'sass:meta';
+@use 'sass:map';
+
+@use '../colors.scss' with (
+  $primary: #3f021a,
+  $good: #010005,
+  $bad: #970934,
+  $fg-map-keys: (),
+  $bg-map-keys: (),
+);
```

Comment:
Bruh, not another theme...

Reply:
I'm so sorry

(It's genuinely really niche, if it's too much overhead we could cut it. I did it cause that's what came to mind when picturing the ui, and cause I wanted to give the system a go)